src/sentry/web/frontend/organization_settings.py

```diff
@@ -57,17 +90,32 @@ def get_form(self, request, organization):
                 'allow_joinleave': bool(organization.flags.allow_joinleave),
                 'enhanced_privacy': bool(organization.flags.enhanced_privacy),
                 'allow_shared_issues': bool(not organization.flags.disable_shared_issues),
+                'scrub_data': bool(organization.get_option('sentry:scrub_data', False)),
```

Comment:
> We need them to simply mean something different.

I'm not entirely sure what you mean. From a logic perspective, I still think these flags are fine and what they evaluate to is correct. I feel like you just want the UI to be reworded. Since as you point out, it's not useful to have a "Disabled for all projects". The act of enabling, dictates this down to all the projects. When not enabled, projects have the liberty to choose.